scripts/devbox-init-hook.sh

```diff
@@ -34,8 +21,7 @@ if [[ ! -f "/usr/bin/envsubst.old" ]]; then
 fi
 
 # Use build location by default
-if [[ ! -e "$INSTALL_DIR/clusterawsadm" ]]; then
+if [[ ! -L "$INSTALL_DIR/clusterawsadm" ]]; then
     echo "Linking [$SRC_DIR/bin/clusterawsadm] [$INSTALL_DIR/clusterawsadm]"
-    mkdir -p "$INSTALL_DIR"
-    ln -s "$SRC_DIR/bin/clusterawsadm" $INSTALL_DIR/clusterawsadm
+    sudo ln -s "$SRC_DIR/bin/clusterawsadm" $INSTALL_DIR/clusterawsadm
```

Comment:
Similar to the previous points i wonder if we should just to `make clusterawsadm` to get the binary built into the `bin` folder?